### ISSUE TITLE
Different IDs used for marking doctors on/off duty

### ISSUE THREAD

**Issue by brianyenna** [brianyenna/pe#5]
When marking a doctor as on duty, the ENTRY_ID is used, but when marking the doctor as off duty the number of the pane on the "Now Serving" section of the UI is used instead of the doctor's ENTRY_ID. 

To be more specific, perhaps an example might be better: 
1. To mark Dr. David Li as on duty, i run `onduty 5`

![Screen Shot 2019-11-15 at 4.48.45 PM.png](https://raw.githubusercontent.com/brianyenna/pe/master/files/ad9e31bb-d208-404f-99fb-257935f38e24.png)

2. To mark Dr. David Li as off duty, i run `offduty 3`, instead of `offduty 5`, because Dr. David Li's ID in the "Now Serving" queue is 3. Running `offduty 5` gives me an error message: "The index provided is invalid."

![Screen Shot 2019-11-15 at 4.49.03 PM.png](https://raw.githubusercontent.com/brianyenna/pe/master/files/68214972-363a-4fc9-8ce9-8d272205b65e.png)

To summarise, the User Guide is not clear on what ON_DUTY_DOCTOR_ENTRY_ID refers to. 

![Screen Shot 2019-11-15 at 4.44.10 PM.png](https://raw.githubusercontent.com/brianyenna/pe/master/files/6c576219-a489-4f3b-bb67-29695bc084f4.png)

This is very confusing for users as there are several indices to keep track.
